rasa/core/policies/two_stage_fallback.py

```diff
@@ -76,20 +81,32 @@ def __init__(
         self.fallback_nlu_action_name = fallback_nlu_action_name
         self.deny_suggestion_intent_name = deny_suggestion_intent_name
 
+    @classmethod
+    def validate_against_domain(
+        cls, ensemble: Optional["PolicyEnsemble"], domain: Optional[Domain]
+    ) -> None:
+        if ensemble is None:
+            return
+
+        for p in ensemble.policies:
+            if isinstance(p, cls):
+                if (
+                    domain is None
+                    or p.deny_suggestion_intent_name not in domain.intents
+                ):
+                    raise InvalidDomain(
+                        "The intent '{}' must be present in the "
+                        "domain file to use the "
+                        "`TwoStageFallbackPolicy`."
```

Comment:
I guess this can go in one line, right?

Reply:
Black keeps reformatting it in weirder ways. This seems like the cleanest of them.